README.md

```diff
@@ -2,101 +2,31 @@
 
 # Sablier Subgraphs and Indexers
 
-Sablier relies on specific dependencies to source data or manage off-chain flows. All of these are either public or
-fully open-source, so feel free to roam around and suggest improvements or optimizations where needed.
+Sablier uses a combination of The Graph subgraphs and Envio indexers as a data API.
 
-As an alternative to reading data from the contracts or listening to onchain events, we use a set of subgraphs and
-indexers. These act as a middleware between the chain and our interfaces and allow for caching, formatting and querying
-data.
+In-depth documentation is available at [docs.sablier.com](https://docs.sablier.com/api/overview). For the list of
+endpoints, see [this](https://docs.sablier.com/api/lockup/endpoints).
 
-For every The Graph subgraph you will find a dedicated Envio indexer that mirrors its functionality and can be used as a
-fallback or as a completely alternative query engine.
+## For contributors
 
-## The Graph Subgraphs
+### Adding a new chain
 
-Read more about The Graph [here](https://thegraph.com/docs/en/).
+Create a dedicated configuration, and add it in the [Envio bundles](./packages/constants/src/bundles/) and provide a
+deployment script for the subgraphs.
 
-Sablier supports multiple chains for which we've deployed subgraphs at either the Hosted Network level or within the
-Decentralized Network (see Ethereum, Arbitrum or Polygon's endpoints).
+### Tracking a new address
 
-Subgraphs are configured using `yarn`.
+See the [`constants`](./packages/constants) folder. Specify the chain, append the contract, and run the codegen steps to
```

Comment:
"...run the codegen steps..."
Might be helpful to explicitly say which codegen tool to run (e.g., graph codegen or a script).
Suggestion: "…run the codegen steps (e.g., yarn codegen in the relevant package)..."